### ISSUE TITLE
No README

### ISSUE THREAD

**Issue by subhojit777** [hoaproject/File#28]
Problem:
I was looking for a Composer package that allows you to create file from a template, and I came across https://packagist.org/packages/hoa/file. I was surprised that this package has a huge download count but there is no README OR any documentation for this package. This documentation https://hoa-project.net/En/Literature/Hack/File.html is not working.

Any newbie wouldn't be able to know what it does, other than just a vague idea that it does *something* with files.

Propsosed resolution:
This Composer package would be more helpful if there is a good documentation. Because,

**Comment by Hywan** [comment]
I agree with you. This is something I am planning to do really soon. Thanks for the clear bug report!
